specifyweb/frontend/js_src/lib/components/ExportFeed/Editor.tsx

```diff
@@ -340,6 +341,16 @@ function ResourcePicker({
   );
 }
 
+const specifyUserTypeSearch: TypeSearch = {
+  displayFields: undefined,
+  table: tables.SpecifyUser,
+  searchFields: [[getField(tables.SpecifyUser, 'name')]],
```

Comment:
this is not safe - you should not be accessing `tables` in global scope as it's only defined after the datamodel is fetched

it might work right now if this file does not get imported before datamode is fetched, but it may randomly stop working in the future

solutions:
1. use f.store() around entire object - this is what we done in other places - read the comment above f.store() as well as look at some usages to see how that fixes it
2. OR move this object into useMemo inside of UserPicker (but 1st solution is probably better)

Reply:
Fixed!